src/components/PictureCardList.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Picture from './PictureCard';
+import style from '../styles/Films.module.css';
+
+const PicturesList = (props) => {
+  const { pictures } = props;
+
+  return (
+    <section className={style.picturesList}>
+      {pictures.map((picture) => (
+        <Picture picture={picture} key={picture.date} />
+      ))}
+    </section>
```

Comment:
- Hey, great job here also, however, According to the project requirement, you should implement search interactivity where users can search for a particular item and the search should return what match the user's input. I don't seem to see it currently in the app as this is a core feature in this project to have it approved. Please let's add this missing part of the project requirement. 


![search](https://user-images.githubusercontent.com/59397159/204937580-2a2943a6-36ab-4385-8e0d-eeca88c38135.png)


 - That's just a tip on how to add it from the image, how you intend to add it is up to you, but please try and follow the design layout as much as you can. Here's the requirement below:


![Screenshot from 2022-12-01 01-19-42](https://user-images.githubusercontent.com/59397159/204937749-d92ee4f8-bede-4a2a-88ba-e9a6afe67211.png)

Reply:
@zieeco Hi, can you please review this, because there are numerous projects, that do not include any kind of filter, and got approved

Reply:
In my case, it is already filtered bye the API

Reply:
Hi @MattGomb , I don't know about that, but if you kindly check the project requirement, you'd see it mentioned. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Reply:
Filter functionality added to project